twitter.py

```diff
@@ -0,0 +1,87 @@
+"""
+This file gets the mentions on twitter and creates a spotify playlist then retweets it
```

Comment:
module, not file

Reply:
changed file to module in this commit [here](https://github.com/GaelGil/apiPractice/commit/24d5855d83265e21000dd2b174bee22fdb6850ea)